doc/network.tex

```diff
@@ -257,39 +258,38 @@ \subsubsection{Network connectivity}\label{network-connectivity}
 
 \paragraph{Participate as a user who has delegated}
 
-As a Daedalus home user with my stake delegated to other users I would like to
-join the Cardano network so I can participate in the network.
+As a Daedalus home user with my stake delegated to other users I would like to join the Cardano network
+so I can participate in the network.
 \begin{itemize}
-\item The system be designed to provide this user segment with the ability
-      to catch up and keep up with the blockchain without having to do any local
-      network configuration.
-\item The system be designed to provide this user segment with the ability to
+\item The system must be designed to provide this user segment with the ability
+      to catch up and keep up with the blockchain without having to do any local network configuration.
+\item The system must be designed to provide this user segment with the ability to
       continuously find and maintain a discovery of a sufficient number of
       other network participants that have reasonable connectivity.
-\item The system be designed to provide this user segment with the ability to
+\item The system must be designed to provide this user segment with the ability to
       find and maintain a minimum of 3 other network participants to maintain
       connectivity with performance that is sufficient to catch up with the
       blockchain.
 \item The system design will take into account that this user will probably be
       behind a firewall.
-\item Users in the segment can be defined by as having all their stake
+\item Users in the segment can be defined by having all their stake
       delegated to other network participants.
```

Comment:
as such they will never be selected as a slot leader (i.e required to generate a block)